src/controllers/mainController.ts

```diff
@@ -747,9 +749,11 @@ export default class MainController implements vscode.Disposable {
             if (editor.document.getText(selectionToTrim).trim().length === 0) {
                 return;
             }
+            vscode.commands.executeCommand('setContext', 'mssql.isQueryRunning', true);
```

Comment:
where is it set to false again once the query is completed?

Reply:
You're right about that, it needs to be added after it.

Reply:
The button disappears after it's run, but it reappears once you click around the area.